plugin-api.d.ts

```diff
@@ -1418,6 +1418,8 @@ interface InstanceNode extends DefaultFrameMixin, VariantMixin {
   scaleFactor: number
   readonly exposedInstances: InstanceNode[]
   isExposedInstance: boolean
+  readonly overrides: NodeChangeProperty[]
+  resetOverrides(): undefined
```

Comment:
This should be `void` like this: https://github.com/figma/plugin-typings/blob/master/plugin-api.d.ts#L924.

`void` tells typescript that no one should use the return value from this function.